docs/apps/components/button.md

```diff
@@ -0,0 +1,83 @@
+# Button
+
+## Description
+
+A button can be used to trigger an event when it's clicked.
+
+Buttons are typically used to submit forms, navigate between pages, or perform other actions.
+
+<img src= "/apps/components/img/button.png">
+
+## Use Case
+
+Some Use Cases are listed as follows:
+
+1. Submit Form
+2. Navigate Through Multi-step Forms/Different Pages (Next, Continue, Previous, Back)
+3. Trigger Specific Actions:
+    * Display or hide additional fields or sections
+    * Perform calculations or validations
+    * Open pop-up windows or dialogs
+    * Reset or Clear Form
+
+4. Trigger Custom Workflows
+
+### Interactivity
+
+**Interactivity** is the ability of a web page or application to respond to user input.
+
+Here are some examples of how interactivity can be implemented for **Button** components:
+
+1. **Submit forms**: A common use of button interactivity is to submit forms to a server for processing. When a user clicks the submit button, the form data is validated and sent to the server, resulting in actions such as saving data, creating accounts, or processing orders.
+
+2. **Display or Hide elements**: Buttons can be used to toggle the visibility of specific elements on the form. For instance, clicking a "Show Advanced Options" button might reveal additional fields or sections of the form, providing more granular control to users.
+
+3. **Perform Data Manipulation or Calculations**: It's useful for real-time validation, dynamic calculations, or updating form fields based on other inputs.
+
+4. **Navigate to Different Pages or Sections**: It allows for multi-step forms, progressive disclosure, or branching based on user choices.
+
+5. **Trigger Animations or Visual Effects**: Buttons can initiate animations or visual effects to enhance the user experience and provide visual feedback. It can include highlighting fields upon clicking, fading in or out elements, or displaying progress indicators.
+
+6. **Open pop-up windows or dialogs**: Buttons can be used to open pop-up windows or dialogs for displaying additional information, collecting supplementary data, or providing context-sensitive guidance.
+
+7. **Trigger custom actions or workflows**: It allows for extending the form builder's functionality and integrating it with external systems.
+
+## API
+
+### Events
+
+| **Name**| **Description**|
+|----------------------|---------------------------------------------------------------------|
+| **onClick (element)**| Event handler that's triggered when the button is clicked. This event handler can be used to perform any number of actions, such as submitting the form, redirecting the user to a different page, or displaying an error message|
+
+### Methods
+
+| **Name**| **Description**|
+|----------------------|---------------------------------------------------------------------|
+|**this.addClassName(fields, disabled)**|Add style class to form item|
```

Comment:
Please separate the parameters from the method name and add them in their own column. There we can add more details to the parameters. e.g (fields: string[], disabled: boolean) in the parameters column. This would give the much neded description of the parameters. Here is an example: https://element-plus.org/en-US/component/tree-v2.html#treev2-method

Reply:
done, also making changes with the other components